internal/cloudapi/v2/errors.go

```diff
@@ -265,8 +264,8 @@ func (s *Server) HTTPErrorHandler(echoError error, c echo.Context) {
 	sec, ok := he.Message.(ServiceErrorCode)
 	if !ok {
 		// No service code was set, so Echo threw this error
-		doResponse(apiErrorFromEchoError(he), c)
+		doResponse(apiErrorFromEchoError(he), c, nil)
```

Comment:
Why nil and not he.Internal as well?

Reply:
oh right, echo can also set `Internal`, fixing.